library/src/webapp/skin/neo-ox/portal.css

```diff
@@ -50,6 +50,27 @@ body.portalBody {
     right:3px;
     vertical-align: middle;
 }
+.oxford-talks{
+    background-image: url(/library/image/silk/fff-sprites/images/oxford-talks.png) !important;
+}
+
+.researcher-training{
+    background-image: url(/library/editor/ckextraplugins/researcher-training-tool/icons/researcher-training-tool.png) !important;
+}
+
+.solo{
+	background-image: url(/library/editor/ckextraplugins/solo-citation/icons/solo-citation.png) !important;
+}
+
+.oxam{
```

Comment:
It's better to try to have slightly more specific CSS classes, so it's clearer where they should be used, but also less likely to clash with other elements.